resources/creatives/models/creative_writable.yml

```diff
@@ -2,8 +2,6 @@ oneOf:
   - allOf:
       - title: Postcard Creative
         required:
-          - front
```

Comment:
I have concerns about removing the asset fields as required fields from the documentation as that might cause some users to create creatives without assets and then go through the campaign creation process and end up with blank mailpieces 😬

Reply:
Yeah I'd stick with making these required and move to making them required on the API side.

Reply:
@julielinter Ah, got it. I wasn't sure how to deal with that, because they don't appear to be formally required, but I'll add the requirements back for now